autoname_workspaces.py

```diff
@@ -91,6 +92,9 @@ def rename_workspaces(i3):
     for ws_index, workspace in enumerate(i3.get_tree().workspaces()):
         ws_info = ws_infos[ws_index]
 
+        if ws_info != focused_workspace(i3):
+            continue
```

Comment:
What happens if a window opens or closes on a workspace that isn't currently focused? For example, sometimes I have a software update program running on a hidden workspace that closes itself when done.